rlkit/core/rl_algorithm.py

```diff
@@ -515,7 +390,15 @@ def obtain_eval_paths(self, deterministic=False):
     def collect_paths(self, idx, epoch, run):
         self.task_idx = idx
         self.env.reset_task(idx)
-        paths = self.obtain_eval_paths(deterministic=False)
+
+        # paths = self.obtain_eval_paths(deterministic=False)
+        self.reset_posterior()
+        paths, _ = self.eval_sampler.obtain_samples(num_samples=self.num_steps_per_eval,
+                                                    resample=self.resample_z)
+        if self.sparse_rewards:
```

Comment:
I think this is outdated, see https://github.com/katerakelly/rlkit/pull/5/commits/9aaf099a4ecc333c73dbcf28c686f9498f2024de

Reply:
looks liek this isn't outdated yet. It's being used for logging in eval, which is still just collecting whatever field is called rewards, so I'll just delete this and change it in the eval part

Reply:
that's what I meant, that the env sparsifying the rewards retroactively is outdated

Reply:
done